src/llmcompressor/modifiers/quantization/calibration.py

```diff
@@ -51,10 +51,9 @@ def initialize_observer(
 
     quantization_args = getattr(quantization_scheme, arg_name, None)
     # dont need observers for dynamic
-    if quantization_args and not quantization_args.dynamic:
-        observer = quantization_args.get_observer()
+    if quantization_args is not None and not quantization_args.dynamic:
         observer = Observer.load_from_registry(
-            observer, quantization_args=quantization_args
+            quantization_args.observer, quantization_args=quantization_args
```

Comment:
I think we should be consistent in how we're fetching the observer - either use the get_observer method or remove it and do it how you're doing it here.

Reply:
Personally in favor of removing the `get_observer` now that observer refactor work is done

https://github.com/vllm-project/llm-compressor/pull/939